docs/MockFunctions.md

```diff
@@ -33,15 +33,15 @@ how the function has been called is kept. The `.mock` property also tracks the
 value of `this` for each call, so it is possible to inspect this as well:
 
 ```javascript
-var myMock = jest.fn();
+const myMock = jest.fn();
 
-var a = new myMock();
-var b = {};
-var bound = myMock.bind(b);
+const a = new myMock();
+const b = {};
+const bound = myMock.bind(b);
 bound();
 
 console.log(myMock.mock.instances);
-> [ <a>, <b> ]
+// > [ <a>, <b> ]
```

Comment:
@dmitriiabramov The syntax of `// >` itself is quite different from the other files, and is only used in this one MockFunctions.md file. Adding comment is necessary otherwise eslint breaks.